common-content/en/module/how-our-curriculum-works/no-lectures/index.md

```diff
@@ -0,0 +1,9 @@
++++
+title="No lectures"
+headless="true"
+time= 30
+[objectives]
+    1="Explain why we don't lecture at CYF"
++++
+
+At CYF, we don’t lecture trainees during live sessions.
```

Comment:
```suggestion
We don’t lecture trainees during live sessions.
```

Also you might use some stuff from **https://docs.google.com/document/d/1xJSTOCWAm4nOvDR8lppG9EKHVbiUR8-A4nt9iI_tLLs/edit?usp=sharing**